src/build-scripts/gh-installdeps.bash

```diff
@@ -83,7 +84,10 @@ else
         locales wget \
         libopencolorio-dev \
         libopencv-dev \
-        libhdf5-dev
+        libhdf5-dev \
```

Comment:
As an aside, I think libhdf5-dev can be removed. I believe that is a holdover from when we used field3d (which we no longer support).

Reply:
Removed